Movies-MVP/Movies-MVP/Modules/MoviesListTab/Presenter/MovieListPresenter.swift

```diff
@@ -0,0 +1,111 @@
+// MovieListPresenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Presenter for Movies screen
+final class MovieListPresenter: MoviesViewPresenterProtocol {
+    // MARK: - Private Constants
+
+    private enum UrlComponent {
+        static let movieBaseUrlText = "https://api.themoviedb.org/3/"
+        static let moviePath = "movie/"
+    }
+
+    private enum CurrentCategoryOfMovies: String {
+        case popular = "popular?"
+        case topRating = "top_rated?"
+        case upComing = "upcoming?"
+    }
+
+    // MARK: - Public properties
+
+    let movieListUrlString = UrlComponent.movieBaseUrlText + UrlComponent.moviePath
+    var fetchingMore = false
+    var isSearching = false
+    var currentPage = 1
+
+    // MARK: - Private properties
+
+    private let networkService: NetworkServiceProtocol?
+    private var router: RouterProtocol?
+    private weak var view: MoviesListProtocol?
+    private(set) var movies: [Movie]? = []
```

Comment:
```suggestion
    private(set) var movies: [Movie]? = []
```
я хз, но я бы отделил локальные переменные от глобальных, а то они тут вперемешку...

Reply:
Добавил пробел между ними, но не буду этот коммент помечать как решенный